scripts/make_help.py

```diff
@@ -1,57 +1,94 @@
 from pathlib import Path
 from argparse import ArgumentParser
-
-
-YELLOW = "\033[33m"
-BLUE = "\033[34m"
-RESET = "\033[0m"
-
-
-def parse_makefile(file: Path, res: dict[str, list[tuple[str, str]]]):
-    with open(file, "r") as f:
-        contents = f.readlines()
-        contents = map(lambda x: x.strip(), contents)
-        contents = filter(lambda x: x.startswith("## "), contents)
-        contents = map(lambda x: x.lstrip("## "), contents)
-
-        for line in contents:
-            data = line.split(":")
-            assert len(data) == 3
-
-            grp = data[0].strip()
-            cmd = data[1].strip()
-            val = data[2].strip()
-
-            if grp in res:
-                res[grp].append((cmd, val))
-            else:
-                res[grp] = [(cmd, val)]
-
-
-def create_help(files: list[Path]) -> dict[str, list[tuple[str, str]]]:
-    res = dict()
+from typing import Dict, List, Tuple
```

Comment:
From python 3.9, list, tuple etc are already generic types. So no need to use typing: https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc